data-types.md

```diff
@@ -3,4 +3,408 @@ title: Data Types
 toc: false
 ---
 
-Coming soon.
\ No newline at end of file
+CockroachDB supports the followinig data types. Click a type for more details.
+
+Type | Description | Example
+-----|-------------|--------
+[`INT`](#int) | A 64-bit signed integer. | `12345`
+[`DECIMAL`](#decimal) | An exact, fixed-point number. | `DECIMAL '1.2345'`
+[`FLOAT`](#float) | An inexact, floating-point number. | `1.2345`
+[`BOOL`](#bool) | A Boolean value. | `true` 
+[`DATE`](#date) | Year, month, day. | `DATE '2016-01-25'`
+[`TIMESTAMP`](#timestamp) | A date and time pairing. | `TIMESTAMP '2016-01-25 10:10:10'`
+[`INTERVAL`](#interval) | A span of time. | `INTERVAL '2h30m30s'`
+[`STRING`](#string) | A string of characters. | `'a1b2c3'`
+[`BYTES`](#bytes) | A string of binary characters. | `'a1b2c3'`
+
+## `INT`
+
+The `INT` type stores 64-bit signed integers, that is, whole numbers from 9,223,372,036,854,775,808 to 9,223,372,036,854,775,807. 
+
+#### Synonyms 
+
+In CockroachDB, the following are synonyms of `INT` and are implemented identically: 
+
+- `SMALLINT` 
+- `INTEGER` 
+- `INT64` 
+- `BIGINT`
+- `BITS`
```

Comment:
Sorry I didn't get this into this PR. I'll open a new one. But I'm a little confused. It looks to me like `BIT(n)` is really equivalent to `INT`. You can declare a column as `BIT(5)` when creating a table, and it'll show up as `INT(5)` on `show columns from`, but we don't enforce the limit. 

```
root@:26257> create table bits (a bit(5), b bit, c int);
CREATE TABLE
root@:26257> show columns from bits;
+-------+--------+-------+----------------+
| Field |  Type  | Null  |    Default     |
+-------+--------+-------+----------------+
| a     | INT(5) | true  | NULL           |
| b     | INT    | true  | NULL           |
| c     | INT    | true  | NULL           |
| rowid | INT    | false | unique_rowid() |
+-------+--------+-------+----------------+
root@:26257> insert into bits values (123456, 1, 2);
INSERT 1
root@:26257> select * from bits;
+--------+---+---+
|   a    | b | c |
+--------+---+---+
| 123456 | 1 | 2 |
+--------+---+---+
```

Maybe I should list `BIT(n)` as a synonym for `INT` and also have a section on length and state that the limit won't be enforced, similar to what we say for [`STRING`](https://www.cockroachlabs.com/docs/string.html)?